core/src/test/scala/org/apache/spark/rdd/PairRDDFunctionsSuite.scala

```diff
@@ -662,6 +796,67 @@ class PairRDDFunctionsSuite extends SparkFunSuite with SharedSparkContext {
       testPoisson(stratifiedData, false, samplingRate, seed, n)
     }
 
+    def testSplits(
+        stratifiedData: RDD[(String, Int)],
+        splitWeights: Array[scala.collection.Map[String, Double]],
+        seed: Long,
+        n: Int,
+        exact: Boolean): Unit = {
+
+      def countByKey[K, V](xs: TraversableOnce[(K, V)]): Map[K, Int] = {
```

Comment:
Do we need this? We should be able to use `rdd.countByKey` in L824 (`totalCounts`), L835 & L836 (`sampleCounts` and `complementCounts`) below?